### DISCUSSION TITLE
NZBGet v23 Client Release

### DISCUSSION THREAD

**Discussion by luckedea** [nzbgetcom/nzbget#186] [Announcements]
The v23 release introduces key updates and new functionalities aimed at enhancing extension management, increasing compatibility, and extending support across multiple platforms.

[Download NZBGet v23 Client Here](https://nzbget.com/download/)

Release highlights:

**Extension Manager Implementation**

- The Extension Manager now facilitates direct management of extensions, including updates, activation, deactivation, configuration, and removal. Compatibility with the existing extension system is maintained.

**Extension Enhancements**

- Adjustments have been made to ensure popular extensions are compatible with the latest Python versions and libraries, as well as with the Extension Manager.

- A centralized repository for extensions is now available at:[ https://github.com/nzbgetcom/nzbget-extensions](https://github.com/nzbgetcom/nzbget-extensions)

- Updates to internal extensions, such as Email and Logger, and to NZBGet’s proprietary extensions like VideoSort, FakeDetector, and FailureLink, have been completed.

- Work on community extensions continues, with ExtendedUnrar complete, Completion underway, and PasswordDetector and EasySort scheduled for updates.

**Expanded Support Across Platforms**

- QNAP devices now fully supported, with Docker and Synology (including SynoCommunity and SPK packages) enhancements.

- Architectural support extended to arm64, aarch64, MIPS (mipseb, mipsel), PowerPC (ppc6xx, ppc500), and RISC-V.

**General Improvements and Fixes**

- The default connection count per news server is now 8.

- News server certificate check levels can now be customized.

- Numerous minor bug fixes have been implemented.

**GitHub Community Contributions Welcome**

Feedback and contributions are essential for the ongoing development of NZBGet. Engage in testing, provide feedback, and visit the [GitHub page](https://github.com/nzbgetcom/nzbget) to suggest features and participate in discussions.

**Comment by OneCDOnly** [comment]
Hi @luckedea , there's been interest in [including your NZBGet QPKG](https://github.com/OneCDOnly/sherpa/discussions/282) with [sherpa](https://github.com/OneCDOnly/sherpa).

It would replace the existing QPKG, which offers NZBGet 21.1 via Entware.

Let me know if this is something you're happy for me to-do. Cheers! 🤓

**Reply by luckedea** [reply]
@OneCDOnly Thanks for reaching out, yes absolutely, what's needed from us?

**Reply by OneCDOnly** [reply]
Ah, that's the best-part - nothing! 😁 

I'll update the references in sherpa to pull and install your prepared QPKG assets from your latest release. Something for me to do today. 

Thank you. 👍🏽

**Reply by luckedea** [reply]
@OneCDOnly Ping me when done (or if you need anything),I'll link it from our download instructions too.

**Reply by OneCDOnly** [reply]
Will do. Your QPKG works-well, and should be good-to-go “live” in sherpa shortly. 👍

I’ll message back when this happens.

**Reply by OneCDOnly** [reply]
Hi @luckedea, we're all good-to-go here. Your NZBGet QPKG is now available via the [sherpa package manager](https://github.com/OneCDOnly/sherpa). 🤓 

For those new to sherpa, once it's installed on your QNAP NAS, NZBGet installation is done with:
```
sherpa install nzbget
```
I forgot to mention: all sherpa-managed QPKGs are pseudo-signed (they are installed, then sherpa hacks the QTS signing database so QTS thinks each QPKG is properly signed), so there's no-need to "allow packages without a signature" in your local App Center.
